components/dashboard/src/components/UsageView.tsx

```diff
@@ -160,13 +175,67 @@ function UsageView({ attributionId }: UsageViewProps) {
 
     const headerTitle = attributionId.kind === "team" ? "Team Usage" : "Personal Usage";
 
+    const DateDisplay = forwardRef((arg: any, ref: any) => (
+        <div
+            className="px-2 p-1 text-gray-500 bg-gray-100 dark:text-gray-400 dark:bg-gray-800 rounded-md cursor-pointer flex items-center"
+            onClick={arg.onClick}
+            ref={ref}
+        >
+            <div className="font-medium">{arg.value}</div>
+            <div>
+                <svg
+                    width="20"
+                    height="20"
+                    fill="currentColor"
+                    xmlns="http://www.w3.org/2000/svg"
+                    onClick={arg.onClick}
+                    ref={ref}
+                >
+                    <path
+                        fillRule="evenodd"
+                        clipRule="evenodd"
+                        d="M5.293 7.293a1 1 0 0 1 1.414 0L10 10.586l3.293-3.293a1 1 0 1 1 1.414 1.414l-4 4a1 1 0 0 1-1.414 0l-4-4a1 1 0 0 1 0-1.414Z"
+                    />
+                    <title>Change Date</title>
+                </svg>
+            </div>
+        </div>
+    ));
+
     return (
         <>
             <Header
-                title={headerTitle}
-                subtitle={`${new Date(startDateOfBillMonth).toLocaleDateString()} - ${new Date(
-                    endDateOfBillMonth,
-                ).toLocaleDateString()} (updated every 15 minutes).`}
+                title={
+                    <div className="flex items-baseline">
+                        <h1 className="tracking-tight">{headerTitle}</h1>
+                    </div>
+                }
+                subtitle={
+                    <div className="tracking-wide flex mt-3 items-center">
+                        <h2 className="mr-2">Showing usage from </h2>
+                        <DatePicker
+                            selected={startDate.toDate()}
+                            onChange={(date) => date && setStartDate(dayjs(date))}
+                            selectsStart
+                            startDate={startDate.toDate()}
+                            endDate={endDate.toDate()}
+                            maxDate={endDate.toDate()}
+                            customInput={<DateDisplay />}
+                            dateFormat={"MMM d, yyyy"}
+                        />
+                        <h2 className="mx-2">to</h2>
+                        <DatePicker
+                            selected={endDate.toDate()}
+                            onChange={(date) => date && setEndDate(dayjs(date))}
+                            selectsEnd
+                            startDate={startDate.toDate()}
+                            endDate={endDate.toDate()}
+                            minDate={startDate.toDate()}
+                            customInput={<DateDisplay />}
+                            dateFormat={"MMM d, yyyy"}
+                        />
```

Comment:
**question**: Could we easily drop the triangle on calendar expand? Feels like the only remaining design issue.

| Light theme | Dark theme |
|-|-|
| <img width="359" alt="triangle-light" src="https://user-images.githubusercontent.com/120486/201178386-498f37d8-7c0f-484c-a7e2-809263eb6e6d.png"> | <img width="359" alt="triangle-dark" src="https://user-images.githubusercontent.com/120486/201178394-c70da96c-8186-42bf-b9fc-753b3536c73a.png"> |